parts/linux/cloud-init/artifacts/components.json

```diff
@@ -42,13 +42,13 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cni:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.5.32",
```

Comment:
we still need 1.5.32, it is what's currently used in AKS prod

Reply:
sure. @rbtr can we add 1.6.5 in the same PR? the tag is going to be released today & we were wondering if that could have any issues to add both versions for 1.5 & 1.6 in the same PR

Reply:
you need to release it first, but otherwise no issue here. we just need to keep N and N+1 version for each release train